### PR TITLE
Add capability to choose Add or Delete item.

### PR DESCRIPTION
**Pull request by scottdangelo**
Add the capability to choose which item is added or deleted from
the shopping cart.
Also, improve output of List shopping cart

### CODE REVIEW

File: ?
Comment (rhagarty) [inline ?:1]: item['cart_number'] is already a str

**Review by rhagarty — CHANGES_REQUESTED** [review]
small nit

File: ?
Comment (scottdangelo) [reply ?:1]: OK. I just changed the name from 'item' to 'cart_number' to make it more clear. Your commit was the one that originally cast item['item'] as a string...

Comment (stevemar) [reply ?:1]: fixed it up anyway

**Review by stevemar — APPROVED** [review]
+2, LGTM, fixed pep8 error

**Review by rhagarty — APPROVED** [review]
looks good

**Pull request merged**